extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js

```diff
@@ -0,0 +1,53 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.kargo', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.slotsContext',
+	'ext.wikia.adEngine.wad.babDetection'
+], function (adContext, slotsContext, babDetection) {
+	'use strict';
+
+	var bidderName = 'kargo',
+		slots = {
+			mercury: {
+				MOBILE_IN_CONTENT: {
+					sizes: [
+						[300, 250]
+					]
+				}
+			}
+		};
+
+	function isEnabled() {
+		return adContext.get('bidders.kargo') && (!babDetection.isBlocking() || adContext.get('opts.wadIL'));
```

Comment:
`return adContext.get('bidders.kargo') && !babDetection.isBlocking()` please. Kargo isn't prepared for recovery